### DISCUSSION TITLE
Get the magnitude and also the error in a certain band (in the observer's frame)

### DISCUSSION THREAD

**Discussion by Codeprey** [mboquien/cigale#60] [Q&A] [answered]
Hello,

If I want to get the magnitude and also the error in a certain band (in the observer's frame), what should I do? Now I can only get these for the rest frame. 

Thank you!

Peter

**Comment by mboquien** [comment]
All the fluxes and uncertainties are estimated in the observed frame, not in the rest frame.

**Reply by Codeprey** [reply]
Pardon, maybe I didn't explain my problem clearly.

By introducing the 'restframe_parameter' module I can calculate the rest-frame color (e.g., u'-r') or the luminosity (e.g., L_g'), and the corresponding errors (which are all shown in the result files), this is very helpful for the host galaxy analysis.

But I also want to get some magnitudes (or luminosity) and the errors in the observed frame, e.g., magnitudes in Swift UVOT bands, in order to get the differential magnitude and error for a transient. This is important for the case that we have obtained some photometric data for the ongoing nuclear transients, while the host galaxy magnitudes in these bands are currently unavailable, so we must calculate synthetic magnitudes and errors to estimate the contribution of their host galaxies.

Thanks!
Peter

**Reply by mboquien** [reply, marked as answer]
I would just add the relevant bands to the `bands` parameters of the `pdf_analysis` module.

**Reply by Codeprey** [reply]
Thank you very much!
